Python/Git_Repo_Spam_Aggregator/README.md

```diff
@@ -0,0 +1,36 @@
+# Find Spam/Invalid Pull Request across GitHub-Repositries
+
+The script retrieves all the spam/invalid PRs from all the Github repositories provided in repositories.json and prints in the command line. The script captures the following data points
```

Comment:
- Replace PRs with Pull Requests
- Put back-tick around `repositories.json` 
- Put full stop at the end of the sentence